hknweb/admin.py

```diff
@@ -0,0 +1,47 @@
+from django.contrib import admin
+from django.contrib.auth.models import User, Group
+from django.contrib.auth.admin import UserAdmin
+from django.conf import settings
+
+# Unregister the provided model admin
+admin.site.unregister(User)
+
+# Register out own model admin, based on the default UserAdmin
+@admin.register(User)
+class CustomUserAdmin(UserAdmin):
+    list_display = ('username', 'first_name', 'last_name', 'email', 'officer', 'candidate')
+    readonly_fields = ('officer', 'candidate')
+
+    actions = ['add_cand', 'add_officer', 'remove_cand', 'remove_officer']
+
+    def officer(self, user):
+        return 'Y' if user.groups.filter(name=settings.OFFICER_GROUP).exists() else ''
+
```

Comment:
If I return True and False, the admin page will display "True" or "False" instead of checkmarks and Xs, which is hard to read. I don't know how to get it to display the checkmarks; it only does that for boolean fields, not functions returning booleans. So 'Y' and '' is the most readable I can come up with.